dev-aws/kafka-shared-msk/customer-billing/topics.tf

```diff
@@ -116,3 +116,20 @@ resource "kafka_topic" "bex_legacy_invoice_api" {
     "cleanup.policy"    = "delete"
   }
 }
+
+resource "kafka_topic" "public_fulfilment_events" {
+  name               = "bex.public.fulfilment_events"
+  replication_factor = 3
+  partitions         = 10
+  config = {
+    "remote.storage.enable" = "true"
+    "retention.bytes"       = "-1"
+    "retention.ms"          = "-1"
```

Comment:
Do you need infinite retention in dev?

Reply:
good catch, no we do not need this in dev.

Reply:
Sorry, hadn't pushed the changes :man_facepalming: